### ISSUE TITLE
Can't find ColorStateList for windowBackground property

### ISSUE THREAD

**Issue by boswelja** [afollestad/aesthetic#17]
```gradle
java.lang.RuntimeException: Unable to resume activity {com.mnml.music.test/com.mnml.music.ui.activity.MainActivity}: android.content.res.Resources$NotFoundException: Can't find ColorStateList from drawable resource ID #0x7f020057
                                                                       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3644)
                                                                       at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3684)
                                                                       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2896)
                                                                       at android.app.ActivityThread.-wrap11(Unknown Source:0)
                                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1591)
                                                                       at android.os.Handler.dispatchMessage(Handler.java:105)
                                                                       at android.os.Looper.loop(Looper.java:164)
                                                                       at android.app.ActivityThread.main(ActivityThread.java:6535)
                                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                                       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
                                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
                                                                    Caused by: android.content.res.Resources$NotFoundException: Can't find ColorStateList from drawable resource ID #0x7f020057
                                                                       at android.content.res.ResourcesImpl.loadColorStateList(ResourcesImpl.java:894)
                                                                       at android.content.res.Resources.loadColorStateList(Resources.java:1053)
                                                                       at android.content.res.TypedArray.getColor(TypedArray.java:459)
                                                                       at com.afollestad.aesthetic.Util.resolveColor(Util.java:87)
                                                                       at com.afollestad.aesthetic.Util.resolveColor(Util.java:80)
                                                                       at com.afollestad.aesthetic.Aesthetic.colorWindowBackground(Aesthetic.java:476)
                                                                       at com.afollestad.aesthetic.Aesthetic.resume(Aesthetic.java:233)
                                                                       at com.afollestad.aesthetic.AestheticActivity.onResume(AestheticActivity.java:19)
                                                                       at com.mnml.music.ui.activity.MainActivity.onResume(MainActivity.java:369)
                                                                       at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1354)
                                                                       at android.app.Activity.performResume(Activity.java:7058)
                                                                       at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3619)

```
Source code at https://github.com/MnmlOS/Music

**Comment by boswelja** [comment]
This tends to only happen on first launch

**Comment by afollestad** [comment]
Fixed in 0.4.1
